css/main.css

```diff
@@ -120,57 +120,57 @@ body {
   align-items: center;
   gap: 8px;
 }
-.footer{
+
+.footer {
   background: #fff;
   width: 100%;
   border-top-right-radius: 50px;
   padding: 100px 0;
   min-height: 100vh;
 }
 
-.footer .categories{
+.footer .categories {
   width: 80%;
   margin: 48px auto;
 }
 
-.categories .category{
+.categories .category {
   padding: 12px;
 }
 
-.category .language-skill{
+.category .language-skill {
   display: flex;
   justify-content: space-between;
```

Comment:
Every styling that has something in common can be put under **reuse.css**. `display: flex` is repeated multiple times here, you can put all its styling into one

`.project-row .topic-text
.category .language-skill{
display: flex;
}`

check others that have same similarity too